src/openzaak/components/zaken/api/cloud_events.py

```diff
@@ -64,3 +64,50 @@ def send_cloud_event(self, cloud_event: dict[str, Any]) -> None:
         subject=cloud_event["subject"],
         status_code=response.status_code,
     )
+
+
+class CloudEventCreateMixin:
+    def perform_create(self, serializer):
+        # TODO
+        # with conditional_atomic(self.notifications_wrap_in_atomic_block)():
```

Comment:
@danielmursa-dev can you make sure this is also implemented similar to regular notifications? This is needed for e.g. `BesluitInformatieObjectViewSet`. Could you also add tests for this (similar to `gh-819`)?